configs/pytorch.json

```diff
@@ -1,8 +1,14 @@
 {
   "index_name": "pytorch",
   "start_urls": [
+    "https://pytorch.org",
+    "https://pytorch.org/index.html",
     "https://pytorch.org/get-started/",
-    "https://pytorch.org/get-started/locally/"
+    "https://pytorch.org/get-started/locally/",
+    "https://pytorch.org/features",
+    "https://pytorch.org/ecosystem",
```

Comment:
This page is leading to external documentation. We will not crawl content that you do not own.

Reply:
But we own that page itself and the words on it. For example we own the words in the boxes before they are linked externally. Isn't that ok?